<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN5CD94237KD" ignored="0" name="Pega OSF Capture Response API Call Regression Test" tests="2" failures="1" timestamp="2021-04-26T12:22:01 IST" time="65.707" errors="0">
  <testcase name="VerifyOSFCaptureResponseOutcomeAPIcallSuccessfully" time="21.014" classname="Mccm.Pega.OSF.PegaTest.OSFAPICallCaptResponseTest"/>
  <testcase name="VerifyOSFOutcomeAPIAcceptedValidationRunSuccessfully" time="30.675" classname="Mccm.Pega.OSF.PegaTest.PegaOSFOutcomeAcceptedAPIVldTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Mccm.Pega.Inbound.PegaMain.PegaMarketIntrctnHistry.PegaOSFOutcomeAPIAcceptedVlidtion(PegaMarketIntrctnHistry.java:222)
at Mccm.Pega.OSF.PegaTest.PegaOSFOutcomeAcceptedAPIVldTest.VerifyOSFOutcomeAPIAcceptedValidationRunSuccessfully(PegaOSFOutcomeAcceptedAPIVldTest.java:54)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- VerifyOSFOutcomeAPIAcceptedValidationRunSuccessfully -->
</testsuite> <!-- Pega OSF Capture Response API Call Regression Test -->
